File: imobil-project/bin/Teren.dart
import 'imobil.dart';
import 'Proprietar.dart';
import 'Poza.dart';
class Teren extends Imobil
{
  Teren(Proprietar proprietar, String locatie, int dimensiune, Poza poze): super(proprietar,locatie,dimensiune,poze);

  @override
  String toString() {
    StringBuffer sb = new StringBuffer();
    sb.writeln('~~TEREN~~');
    sb.writeln('PROPRIETAR: ' +proprietar.toString());
    sb.writeln('LOCATIE: '+locatie.toString());
    sb.writeln('DIMENSIUNE: '+dimensiune.toString());
    sb.writeln('POZE: '+poze.toString());
    return sb.toString();

  }
}

File: Imobil C#/Imobil/Teren.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Imobil
{
    class Teren: Imobil
    {
        public Teren(Proprietar proprietar, string locatie, int dimensiune, Poza poze): base(proprietar,locatie,dimensiune,poze)
        {

        }
        public override string ToString()
        {
            StringBuilder sb = new StringBuilder();
            sb.AppendLine("~~TEREN~~");
            sb.AppendLine($"PROPRIETAR: {proprietar}");
            sb.AppendLine($"LOCATIE: {locatie}");
            sb.AppendLine($"DIMENSIUNE: {dimensiune}");
            sb.AppendLine($"POZE: {poze}");
            return sb.ToString();
        }
    }
}
